src/server/game/Entities/Player/Player.cpp

```diff
@@ -9548,7 +9548,18 @@ uint8 Player::FindEquipSlot(ItemTemplate const* proto, uint32 slot, bool swap) c
             // suggest offhand slot only if know dual wielding
             // (this will be replace mainhand weapon at auto equip instead unwonted "you don't known dual wielding" ...
             if (CanDualWield())
-                slots[1] = EQUIPMENT_SLOT_OFFHAND;
+            {
+                if (CanTitanGrip())
+                {
+                    Item* mainItem = GetItemByPos(INVENTORY_SLOT_BAG_0, EQUIPMENT_SLOT_MAINHAND);
+                    if (mainItem && mainItem->GetTemplate()->CanBeDualWieldedWithTitansGrip() && proto->CanBeDualWieldedWithTitansGrip())
+                    {
```

Comment:
brackets not needed

Reply:
Yes, when there is only one line below or the completion of the line above (making it the same line), it is also easier to read the code without the redundant curly brackets (line 9556 + 9558) (in addition to being redundant codestyle-wise).